sphinx_gitstamp/__init__.py

```diff
@@ -30,14 +31,8 @@ def page_context_handler(app, pagename, templatename, context, doctree):
     if g is None:
         # We have already errored about this
         pass
-    fullpagename = pagename
-    docsrc = ""
-    try:
-        docsrc = app.confdir + "/"
-        if docsrc != "/":
-            fullpagename = docsrc + pagename
-    except KeyError:
-        pass
+
+    fullpagename = Path(app.confdir) / pagename
```

Comment:
Might be ever-so-slightly quicker:
```suggestion
    fullpagename = Path(app.confdir, pagename)
```

Reply:
Ah, thanks!!